From 9d41cf4fa0e2e914f549f4cd2c9a92b51d7a0b82 Mon Sep 17 00:00:00 2001
From: Tony Xie <tony.xie@rock-chips.com>
Date: Wed, 2 Jan 2019 05:19:57 -0500
Subject: [PATCH] UPSTREAM: mfd: rk808: Remove the id_table

Remove the id_table because it's not used.

Change-Id: I406d5bd9609652cfee83af2ab7571af4e06f3aff
Signed-off-by: Tony Xie <tony.xie@rock-chips.com>
Signed-off-by: Lee Jones <lee.jones@linaro.org>
(cherry picked from commit 54349b3ce82e3369a7c7b2960f19c1bea876c5fb)
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 drivers/mfd/rk808.c | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/drivers/mfd/rk808.c b/drivers/mfd/rk808.c
index cc4643e79f05..2361f1a3bce9 100644
--- a/drivers/mfd/rk808.c
+++ b/drivers/mfd/rk808.c
@@ -1049,15 +1049,6 @@ static const struct dev_pm_ops rk808_pm_ops = {
 	.resume =  rk808_resume,
 };
 
-static const struct i2c_device_id rk808_ids[] = {
-	{ "rk805" },
-	{ "rk808" },
-	{ "rk816" },
-	{ "rk818" },
-	{ },
-};
-MODULE_DEVICE_TABLE(i2c, rk808_ids);
-
 static struct i2c_driver rk808_i2c_driver = {
 	.driver = {
 		.name = "rk808",
@@ -1066,7 +1057,6 @@ static struct i2c_driver rk808_i2c_driver = {
 	},
 	.probe    = rk808_probe,
 	.remove   = rk808_remove,
-	.id_table = rk808_ids,
 };
 
 module_i2c_driver(rk808_i2c_driver);
-- 
2.35.3

